### PR TITLE
Not supported: Explicit references to apps and custom claims

### PR DESCRIPTION
**Pull request by Tennyx**
There may be a more efficient way to call attention to this besides repeating the same note twice - but I think it should be stated that the explicit reference to apps will not work when creating custom claims for OIDC/OAuth. This recently tripped up a customer of mine. If you ping me on Okta slack I can share a convo I had about this as well internally. Thanks!

## Description:
- **What's changed?** 
- **Is this PR related to a Monolith release?** 

### Resolves:

* [OKTA-XXXXXX](https://oktainc.atlassian.net/browse/OKTA-XXXXXX)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/reference/okta-expression-language/index.md [hunk 1]
```diff
@@ -30,6 +30,9 @@ In addition to an Okta user profile, all users have a separate Application user
 | `$appuser.$attribute` | `$appuser` explicit reference to specific app<br>`$attribute` the attribute variable name  | zendesk.firstName<br>active_directory.managerUpn<br>google_apps.email |
 | `appuser.$attribute`  | `appuser` implicit reference to in-context app<br>`$attribute` the attribute variable name | appuser.firstName                                                     |
 
+> **Note:** Explicit references to apps are not supported for custom OAuth/OIDC claims.
+>
+
 **Note:** The application reference is usually the `name` of the application, as distinct from the `label` (display name).  (See [Application properties](/docs/reference/api/apps/#application-properties).)  If your organization has configured multiple instances of the same application, the names of the later instances are differentiated by a randomly assigned suffix, e.g. `zendesk_9ao1g13`.  The name of any specific App instance can be found in the Profile Editor, where it is shown in lighter text beneath the App's label.
```
Comment (jakubvul-okta) [inline packages/@okta/vuepress-site/docs/reference/okta-expression-language/index.md:36]:
```suggestion
> **Note:** The application reference is usually the `name` of the application, as distinct from the `label` (display name).  (See [Application properties](/docs/reference/api/apps/#application-properties).)  If your organization has configured multiple instances of the same application, the names of the later instances are differentiated by a randomly assigned suffix, e.g. `zendesk_9ao1g13`.  The name of any specific App instance can be found in the Profile Editor, where it is shown in lighter text beneath the App's label.
```

File: packages/@okta/vuepress-site/docs/reference/okta-expression-language/index.md [hunk 2]
```diff
@@ -30,7 +30,10 @@ In addition to an Okta user profile, all users have a separate Application user
 | `$appuser.$attribute` | `$appuser` explicit reference to specific app<br>`$attribute` the attribute variable name  | zendesk.firstName<br>active_directory.managerUpn<br>google_apps.email |
 | `appuser.$attribute`  | `appuser` implicit reference to in-context app<br>`$attribute` the attribute variable name | appuser.firstName                                                     |
 
-**Note:** The application reference is usually the `name` of the application, as distinct from the `label` (display name).  (See [Application properties](/docs/reference/api/apps/#application-properties).)  If your organization has configured multiple instances of the same application, the names of the later instances are differentiated by a randomly assigned suffix, e.g. `zendesk_9ao1g13`.  The name of any specific App instance can be found in the Profile Editor, where it is shown in lighter text beneath the App's label.
+> **Note:** Explicit references to apps are not supported for custom OAuth/OIDC claims.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/reference/okta-expression-language/index.md:33]:
```suggestion
> **Note:** Explicit references to apps aren't supported for custom OAuth/OIDC claims.
```

File: packages/@okta/vuepress-site/docs/reference/okta-expression-language/index.md [hunk 3]
```diff
@@ -30,7 +30,10 @@ In addition to an Okta user profile, all users have a separate Application user
...
+>
+
+> **Note:** The application reference is usually the `name` of the application, as distinct from the `label` (display name).  (See [Application properties](/docs/reference/api/apps/#application-properties).)  If your organization has configured multiple instances of the same application, the names of the later instances are differentiated by a randomly assigned suffix, e.g. `zendesk_9ao1g13`.  The name of any specific App instance can be found in the Profile Editor, where it is shown in lighter text beneath the App's label.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/reference/okta-expression-language/index.md:36]:
```suggestion
> **Note:** The application reference is usually the `name` of the application, as distinct from the `label` (display name).  (See [Application properties](/docs/reference/api/apps/#application-properties).)  If your organization has configured multiple instances of the same application, the names of the later instances are differentiated by a randomly assigned suffix, for example: `zendesk_9ao1g13`.  You can find the name of any specific App instance in the Profile Editor, where it is shown in lighter text beneath the App's label.
```

**Review by susanharper-okta — COMMENTED** [review]
couple minor things

File: packages/@okta/vuepress-site/docs/reference/okta-expression-language/index.md [hunk 4]
```diff
@@ -52,6 +55,9 @@ In addition to referencing user attributes, you can also reference App propertie
 | `$app.$attribute` | `$app` explicit reference to specific app instance<br>`$attribute` the attribute variable name  | office365_app.domain<br>zendesk_app.companySubDomain |
 | `app.$attribute`  | `app` implicit reference to in-context app instance<br>`$attribute` the attribute variable name | app.domain<br>app.companySubDomain                     |
 
+> **Note:** Explicit references to apps are not supported for custom OAuth/OIDC claims.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/reference/okta-expression-language/index.md:58]:
```suggestion
> **Note:** Explicit references to apps aren't supported for custom OAuth/OIDC claims.
```

**Review by susanharper-okta — APPROVED** [review]
LGTM

**Pull request merged**